xarray/core/dataset.py

```diff
@@ -2742,6 +2826,42 @@ def head(
             The keyword arguments form of ``indexers``.
             One of indexers or indexers_kwargs must be provided.
 
+        Examples
+        --------
+        >>> dates = pd.date_range(start="2023-01-01", periods=5)
+        >>> pageviews = [1200, 1500, 900, 1800, 2000]
+        >>> visitors = [800, 1000, 600, 1200, 1500]
+        >>> dataset = xr.Dataset(
+        ...     {
+        ...         "pageviews": (("date"), pageviews),
+        ...         "visitors": (("date"), visitors),
+        ...     },
+        ...     coords={"date": dates},
+        ... )
+
+        # Retrieve the 3 most busiest days in terms of pageviews and visitors
+
+        >>> busiest_days = dataset.sortby(["pageviews", "visitors"], ascending=False)
+        >>> busiest_days.head(3)
```

Comment:
```suggestion
        >>> busiest_days.head()
```
If you don't pass the integer here, it should default to 5, which will then give a visiably different result to when you explicitly pass 3 below.